src/main/java/se/sundsvall/relations/integration/db/model/RelationEntity.java

```diff
@@ -0,0 +1,85 @@
+package se.sundsvall.relations.integration.db.model;
+
+import static java.time.OffsetDateTime.now;
+import static java.time.temporal.ChronoUnit.MILLIS;
+import static org.hibernate.annotations.TimeZoneStorageType.NORMALIZE;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.PrePersist;
+import jakarta.persistence.PreUpdate;
+import jakarta.persistence.Table;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.TimeZoneStorage;
+import org.hibernate.annotations.UuidGenerator;
+
+@Data
+@Builder(setterPrefix = "with")
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "relation")
+public class RelationEntity {
```

Comment:
Create RelationEntity that contains:

source (type ResourceIdentifierEntity)
target  (type ResourceIdentifierEntity)

The "ResourceIdentifierEntity" (can be renamed to something better?

(according to disussion)